fonz/connection.py

```diff
@@ -257,6 +328,9 @@ def get_dimensions(self, model: str, explore_name: str) -> List[str]:
 
         return response.json()["fields"]["dimensions"]
 
+    @backoff.on_exception(
```

Comment:
Is this to handle errors we are seeing from Looker?

Reply:
Basically it's a retry in case we get a timeout or unexpected error.